software/glasgow/database/jedec.py

```diff
@@ -19,6 +19,47 @@ def jedec_mfg_name_from_bank_num(bank, mfg_num):
         if number == mfg_num:
             return name
 
+def jedec_update_mfg_from_pdf():
```

Comment:
```suggestion
def _jedec_update_mfg_from_pdf():
```

Can you make it private and put at the very end?

Reply:
Can do. 
Moving to the end will just require a re.replace() to replace the array, rather than my current trivial method of timing the end of the file.

Reply:
Oh right I missed that. Maybe it's worth at least marking the autogenerated section as such, then?

Reply:
This was actually pretty easy to do with a `re.sub()`. Now the updating function is private at the end of the file.